roles/openshift_sanitize_inventory/tasks/__deprecations_logging.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
@michaelgugino I'm not really worried about that file going stale since it should be removed when we do our next minor release since we won't support the usage of those variables anymore, we wouldn't want to map them.

For that reason and the fact that this would run before any other plays/roles that could use these variables and we could map it prior to even making it down into the role, it made sense to me to have it in the same location that we are doing inventory sanitization...